package.json

```diff
@@ -1,53 +1,47 @@
 {
   "name": "opendistro-alerting",
-  "version": "1.11.0.2",
+  "version": "1.12.0.0",
   "description": "Kibana Alerting Plugin",
   "main": "index.js",
   "license": "Apache-2.0",
   "homepage": "https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin",
   "kibana": {
-    "version": "7.9.1",
-    "templateVersion": "6.3.3"
+    "version": "7.10.0"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin"
   },
   "scripts": {
     "kbn": "node ../../scripts/kbn",
-    "lint": "../../node_modules/.bin/eslint '**/*.js' -c .eslintrc",
+    "lint": "../../node_modules/.bin/eslint '**/*.js' -c .eslintrc --ignore-path .gitignore",
     "start": "plugin-helpers start",
-    "test:server": "plugin-helpers test:server",
-    "test:browser": "plugin-helpers test:browser",
     "test:jest": "../../node_modules/.bin/jest --config ./test/jest.config.js",
-    "build": "plugin-helpers build"
+    "build": "yarn plugin-helpers build",
```

Comment:
Why does this need yarn prefixed?

Reply:
I will have a try without `yarn` prefixed, originally I just followed their instruction https://github.com/elastic/kibana/tree/master/packages/kbn-plugin-helpers#kbnplugin-helpers

Reply:
The `plugin-helpers` is not a executable file but a script in the new version, but I'm not very familiar with how the script is defined here, will do more research.

Reply:
Adding the `yarn` prefix is required to run a Javascript file, and the `plugin-helpers` is not an executable file now but a JS file. (For others' reference: https://yarnpkg.com/cli/run https://classic.yarnpkg.com/en/docs/package-json/#toc-scripts)